proto/pocket/service/tx.proto

```diff
@@ -10,7 +15,9 @@ service Msg {
 }
 
 message MsgAddService {
```

Comment:
1. Please add a godoc on what this message is for.
2. Please add a `TODO_DOCUMENT(@h5law): This is a key function in making services permissionless, document it's usage and design`